docs/teslajsonpy/teslajsonpy.energy.rst

```diff
@@ -0,0 +1,10 @@
+==========================
```

Comment:
Is the make docs broken? This seems really light. The CI for making docs doesn't really work so we shouldn't rely on it especially given the huge changes you have. I believe your functions/classes look documented enough.

Reply:
Docs are one area I'm not really sure how to update. I noticed we're using sphinx docs and I tried looking through the documentation to figure it out but didn't get very far and could use help with.

Reply:
I fixed an indent and ran `make docs`. I've pushed it to your repo so this should be resolved. Please do fix the failing tests though. `make test`

Reply:
Thanks. I'm showing all the tests are passing with `make test`.

Reply:
Odd, my local is still reporting a test failure but the CI online seems to be ok. I'll merge this in and see if that PR triggers it. Assuming it doesn't, this will be the breaking change for us in teslajsonpy.